### PR TITLE
refactor: define ValueName for all variable namesa in message

### PR DESCRIPTION
**Pull request by sunnyszy**
As we talked about today, I define an Enum called ValueName in message.py for us to unify all sensor/control names in all modules.

@sneakers-the-rat 

- All the interface to GUI is using ValueName Enum now (The interface between the coordinator and controller still needs refactoring). 
- I modified some of your variable names in value.py. You can take a look, and do corrections. I'm not sure what I did is the most convenient way for you to use. I also move value.py into GUI folder because I think a lot of related definitions are specified for GUI (but I'm not sure this is the best approach).

### CODE REVIEW

File: tests/test_controller.py [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
 from vent.coordinator.coordinator import get_coordinator
 from vent.controller.control_module import get_control_module
 
 
-@pytest.mark.parametrize("control_setting_name", [ControlSettingName.PIP,
-                                                  ControlSettingName.PIP_TIME,
-                                                  ControlSettingName.PEEP,
-                                                  ControlSettingName.BREATHS_PER_MINUTE,
-                                                  ControlSettingName.INSPIRATION_TIME_SEC])
+@pytest.mark.parametrize("control_setting_name", [ValueName.PIP,
```
[the remark this answers is not in the document]
Comment (mschottdorf) [reply tests/test_controller.py:11]: this is done, but not with a list (yet)

File: vent/controller/control_module.py [hunk 2]
```diff
@@ -289,31 +289,31 @@ def set_control(self, control_setting: ControlSetting):
         ''' Updates the entry of COPY contained in the control settings'''
         self._lock.acquire()
 
-        if control_setting.name == ControlSettingName.PIP:
+        if control_setting.name == ValueName.PIP:
```
[the remark this answers is not in the document]
Comment (mschottdorf) [reply vent/controller/control_module.py:292]: Value Names are there now, working on the rest

**Comment by mschottdorf** [comment]
I've tried to resolve the merging conflicts, and merged most of the things discussed here with the main branch (after a prolonged battle with git...).

**Comment by sunnyszy** [comment]
> I've tried to resolve the merging conflicts, and merged most of the things discussed here with the main branch (after a prolonged battle with git...).

Where is your intermediate commit? I can take that to merge the rest of the code.

**Comment by sunnyszy** [comment]
FYI I close this PR because this has been merged partially with master and I'll create a new PR to master.

**Pull request closed without merging**